pkg/mimirpb/mimir.proto

```diff
@@ -80,3 +81,59 @@ message Exemplar {
   double value = 2;
   int64 timestamp_ms = 3;
 }
+
+// This is based on https://github.com/prometheus/prometheus/blob/main/prompb/types.proto
```

Comment:
Do we need to acknowledge this is derived from Prometheus with `Provenance-includes-location`/`Provenance-includes-license` etc. comments at the top of this file?

Reply:
sure